docs/Panorama Paint Rebuild3.md

```
@@ -0,0 +1,237 @@
+- 外部から見た動作は変わらない
+
+---
+
+### Phase 4 — 入力拡張
+
+**目標：ERP画像以外のシーン表現に対応する**
+
+> **方針：技術と型はセットで実装する。**  
+> 深度マップのノードだけ作っても、それを生成する手段がなければワークフローとして完結しない。  
+> 対応する生成技術が揃った時点で、その型と対応ノードをセットで実装する。
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)